sklearn/neural_network/_rbm.py

```diff
@@ -113,12 +113,12 @@ def transform(self, X):
 
         Parameters
         ----------
-        X : {array-like, sparse matrix} shape (n_samples, n_features)
+        X : {ndarray, sparse matrix} of shape (n_samples, n_features)
```

Comment:
There is a few of these left, otherwise LGTM.

Reply:
Please check it now.